android/app/src/main/java/io/cozy/flagship/mobile/MainApplication.java

```diff
@@ -4,6 +4,7 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import sk.kedros.playintegrity.RNGooglePlayIntegrityPackage;
```

Comment:
Why is this needed?

Reply:
> Why is this needed?

Somehow the app wouldn't build without it, had to link manually